packages/react-native/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -170,10 +170,9 @@ export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
    */
   public onAuthEvent(listener?: StitchAuthListener) {
     if (listener) {
-      const auth = this;
       const _ = new Promise(resolve => {
         if (listener.onAuthEvent) {
-          listener.onAuthEvent(auth);  
+          listener.onAuthEvent(this);  
         }
```

Comment:
Just making sure passing the object directly has no impact, shouldnt be any different than assigning it to the variable auth though